docs/en/stack/ml/df-analytics/ml-inference.asciidoc

```diff
@@ -45,3 +45,11 @@ data together with the results are indexed into the destination index.
 Check the {ref}/inference-processor.html[{infer} processor] and 
 {ref}/ml-df-analytics-apis.html[the {ml} {dfanalytics} API documentation] to 
 learn more about the feature.
+
+
+[discrete]
+==== {lang-ident-cap} pre-trained model
```

Comment:
I should know this, but I figure since it's not obvious to me perhaps it won't be obvious to our users either.  Isn't this pre-trained model just a ready-made version of the models described in the "trained models as functions" section? If so, I think this paragraph should just be merged into that section.

Reply:
Yes, basically it is. An OOTB trained model. I moved the text about language identification to the trained models as functions section.